### PR TITLE
Adds Tests

### PR DESCRIPTION
**Pull request by jacobhhamer**
This pull request adds travis testing functionality. It also has more extensive documentation, as well as test data.

### CODE REVIEW

File: tests/test_PCA.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from pca.pca import *
+
```
Comment (vditomasso) [inline tests/test_PCA.py:2]: Do you need to import pytest here?

Comment (jacobhhamer) [reply tests/test_PCA.py:2]: travis is working, so I don't think so

**Comment by kelle** [comment]
your tests look awesome! Do you expect them all to be passing or are you ok with this PR being merged while failing?

**Comment by kelle** [comment]
I see that all the tests passed for Python 3.6 and just something is weird with the Travis setup for Python 2 versions. I think this PR is good to go!

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 language: python
 
 python:
-- 3.6.0
+ - "2.6.0"
```
Comment (kelle) [inline .travis.yml:4]: looks like these two lines are the only problem. try to fix for next PR.

**Review by kelle — APPROVED** [review]
Please fix testing for Python 2.# versions next time.

**Pull request merged**